elytron-jdbc-security tests are unstable

**Describe the bug**
Fails in this [build](https://dev.azure.com/quarkus-ci/quarkus/_build/results?buildId=9286&view=logs): 
```
[ERROR] Tests run: 14, Failures: 6, Errors: 0, Skipped: 0, Time elapsed: 1.018 s <<< FAILURE! - in io.quarkus.elytron.security.jdbc.MultipleDataSourcesTest
[ERROR] testJaxrsGetRoleSuccess  Time elapsed: 0.078 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <200> but was <500>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
```
No further clues are available in the rest of the trace.

@danielpetisme @stuartwdouglas can you please check if you have few secs ?
On my Fedora laptop it passes:
```
[INFO] Tests run: 14, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.494 s - in io.quarkus.elytron.security.jdbc.MultipleDataSourcesTest
```

No, they are still failing on Windows, different test this time:
```
[ERROR] Tests run: 14, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 7.79 s <<< FAILURE! - in io.quarkus.elytron.security.jdbc.BcryptPasswordMapperTest
[ERROR] testJaxrsGetDenyAllWithAuth  Time elapsed: 0.094 s  <<< FAILURE!
```
`MultipleDataSourcesTest` which was failing earlier is passing

I'm seeing the following errors while debugging the `BcryptPasswordMapperTest` in my Windows machine (not sure if it's related): 

```java
Caused by: org.h2.jdbc.JdbcSQLException: Table "TEST_USER" already exists; SQL statement:
CREATE TABLE test_user (
  id INT,
  username VARCHAR(255),
  password_hash VARCHAR(255),
  salt VARCHAR(255),
  iteration_count INT,
  role VARCHAR(255)
) [42101-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:86)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:79)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:58)
	at org.h2.command.CommandContainer.update(CommandContainer.java:102)
	at org.h2.command.Command.executeUpdate(Command.java:261)
	at org.h2.engine.Engine.openSession(Engine.java:227)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/bcrypt-password-mapper/import.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/bcrypt-password-mapper/import.sql
@@ -1,3 +1,5 @@
+DROP TABLE test_user IF EXISTS;
+
 CREATE TABLE test_user (
   id INT,
   username VARCHAR(255),

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/custom-role-decoder/import.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/custom-role-decoder/import.sql
@@ -1,3 +1,5 @@
+DROP TABLE test_user IF EXISTS;
+
 CREATE TABLE test_user (
   id INT,
   username VARCHAR(255),

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/minimal-config/import.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/minimal-config/import.sql
@@ -1,3 +1,5 @@
+DROP TABLE test_user IF EXISTS;
+
 CREATE TABLE test_user (
   id INT,
   username VARCHAR(255),

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-data-sources/permissions.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-data-sources/permissions.sql
@@ -1,7 +1,11 @@
+DROP TABLE test_role IF EXISTS;
+
 CREATE TABLE test_role (
   id INT,
   role_name VARCHAR(255)
 );
+
+DROP TABLE test_user_role IF EXISTS;
 
 CREATE TABLE test_user_role (
   username VARCHAR(255),

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-data-sources/users.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-data-sources/users.sql
@@ -1,3 +1,5 @@
+DROP TABLE test_user IF EXISTS;
+
 CREATE TABLE test_user (
   id INT,
   username VARCHAR(255),

--- a/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-queries/import.sql
+++ b/extensions/elytron-security-jdbc/deployment/src/test/resources/multiple-queries/import.sql
@@ -1,13 +1,19 @@
+DROP TABLE test_user IF EXISTS;
+
 CREATE TABLE test_user (
   id INT,
   username VARCHAR(255),
   password VARCHAR(255)
 );
 
+DROP TABLE test_role IF EXISTS;
+
 CREATE TABLE test_role (
   id INT,
   role_name VARCHAR(255)
 );
+
+DROP TABLE test_user_role IF EXISTS;
 
 CREATE TABLE test_user_role (
   user_id INT,

